### PR TITLE
added VTS disruptor

### PR DESCRIPTION
**Pull request by kshileev**
Change-Id: Ibebfc595c2f9f8463cf1d43832d5f1850ae5b4c2

### CODE REVIEW

File: configs/ha/tc-vts-Tcbr97c.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+- class: lab.scenarios.vts_scenario.VtsScenario
+  delay: 20
+  how-many-networks: 1
+  how-many-servers: 3
+
+- class lab.disruptor.VtcDisruptor
+  delay: 100
+  node-to-disrupt: active-vtc
+  downtime: 100
```
Comment (dratushnyy) [inline configs/ha/tc-vts-Tcbr97c.yaml]: It is good idea to add uptime to

File: configs/ha/tc-vts-Tcbr97c.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  duration: 200
+  period: 2
+
+- class: lab.scenarios.vts_scenario.VtsScenario
+  delay: 20
+  how-many-networks: 1
+  how-many-servers: 3
+
+- class lab.disruptor.VtcDisruptor
+  delay: 100
```
Comment (dratushnyy) [inline configs/ha/tc-vts-Tcbr97c.yaml]: may be we can provide not only duration but also  number of iterations

File: configs/ha/tc-vts-Tcbr97c.yaml (hunk 1 above)
Comment (kshileev) [inline configs/ha/tc-vts-Tcbr97c.yaml]: done

File: configs/ha/tc-vts-Tcbr97c.yaml (hunk 2 above)
Comment (kshileev) [inline configs/ha/tc-vts-Tcbr97c.yaml]: done

**Pull request merged**